examples/example1/parse_rwpod.rb

```diff
@@ -0,0 +1,17 @@
+require_relative '../../lib/crawler'
+
+Crawler.run do
+  configuration do |conf|
+    conf.max_pages 10
+  end
+
+  before :all do
+    visit_to 'https://www.epam.com/'
```

Comment:
Could you please use neutral urls in tests? I'm not sure if pinging a corporate site this way is a good idea.